scio-core/src/main/scala/com/spotify/scio/bigquery/dynamic/package.scala

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2017 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.bigquery
+
+import com.google.api.services.bigquery.model.TableSchema
+import com.spotify.scio.bigquery.types.BigQueryType.HasAnnotation
+import com.spotify.scio.io.Tap
+import com.spotify.scio.util.Functions
+import com.spotify.scio.values.SCollection
+import org.apache.beam.sdk.io.gcp.bigquery.BigQueryIO.Write.{CreateDisposition, WriteDisposition}
+import org.apache.beam.sdk.io.gcp.bigquery.{DynamicDestinations, TableDestination}
+import org.apache.beam.sdk.io.gcp.{bigquery => bqio}
+import org.apache.beam.sdk.values.ValueInSingleWindow
+
+import scala.concurrent.Future
+import scala.reflect.ClassTag
+import scala.reflect.runtime.universe._
+
+/**
+ * BigQuery package for multiple table destinations. Import All.
```

Comment:
You use `multiple` multiple times here, while more it's might not be necessary true, it's more that it's dynamic, maybe you should use `dynamic` instead of `multiple`?

Reply:
makes sense 👍